pkgs/stdenv/generic/check-meta.nix

```
@@ -209,6 +300,59 @@ let
 
       '';
 
+  remediate_problem = pkg: let
+    pkgName = lib.getName pkg;
+
+    fullMessage = problem: (if problem.kind != problem.name then problem.kind + ": " else "") + problem.name + ": " + problem.message;
+    urlMessage = problem: let
+      urls = problem.urls or [];
+    in
+      lib.concatMapStrings (url: "\n  - " + url) urls;
+
+    printProblem = problem: "- " + fullMessage problem + urlMessage problem;
+
+    problems = generateProblems pkg;
+
+    ignorePattern = indentation: lib.pipe problems [
+      (lib.filter ({handler, ...}: handler == "error"))
+      (builtins.map ({name, ...}: ''"${pkgName}"."${name}" = "warn";''))
+      (lib.concatStringsSep ("\n" + indentation))
+    ];
+  in ''
+
+
+    Package problems:
+
+    ${lib.pipe problems [
+      (lib.filter ({handler, ...}: handler == "error"))
+      (builtins.map printProblem)
+      (lib.concatStringsSep "\n")
+    ]}
+
+    You can install it anyway by ignoring its problems, using one of the
```

```suggestion
    You can use it anyway by ignoring its problems, using one of the
```

install means nothing or few in nixpkgs

---

It's the wording all the others use, I just copied it. I'll update it here but leave the others as they are for now.